src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -1063,11 +1063,13 @@ To use a custom JDK path for a command line build, set the 'JavaSdkDirectory' MS
 {1} - NuGet package version</comment>
   </data>
   <data name="XA0141" xml:space="preserve">
-    <value>NuGet package '{0}' version '{1}' contains a shared library '{2}' which is not correctly aligned. See https://developer.android.com/guide/practices/page-sizes for more details</value>
+    <value>NuGet package '{0}' version '{1}' contains a shared library '{2}' ({3}) which is not aligned to the {4} byte boundary. Google will require such alignment in the future. See https://developer.android.com/guide/practices/page-sizes for more details</value>
```

Comment:
This is going to clash with https://github.com/dotnet/android/pull/9559/files#diff-4fb7800ecd2e6254dea359880760d5145fe7d214c9cd45935f897b6f6d0d47daR1066. 

@jonpryor we need to figure out the wording we want.